problems/objects.js

```diff
@@ -151,15 +199,27 @@ const authorScores = () => {};
  * @returns {string} The full name of person with best score.
  */
 
-const bestScore = () => {};
+const bestScore = (submissions) => {// did not understand question
+  for(let i = 0; i < submissions.length; i++) {
+    if(submissions[i]["score"] > submissions[submissions.length -1]["score"]){
+      return submissions[i]["firstName"] + " " + submissions[i]["lastName"]
```

Comment:
Although this function passes the test, unfortunately, it does not work the way we want it to. Consider this object ...

const peopleWithScores = [
      {
        firstName: "Calvin",
        lastName: "Newton",
        score: 13,
      },
      {
        firstName: "Garry",
        lastName: "Mckenzie",
        score: 23,
      },
      {
        firstName: "Leah",
        lastName: "Rivera",
        score: 16,
      },
      {
        firstName: "Sonja",
        lastName: "Moreno",
        score: 3,
      },
      {
        firstName: "Noel",
        lastName: "Bowen",
        score: 10,
      },
    ];

Would you still get the person with the highest score?